Games/baj.2m.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+# <bitbar.title>BAJ</bitbar.title>
+# <bitbar.version>v1.0</bitbar.version>
+# <bitbar.author>Justin Honold</bitbar.author>
+# <bitbar.author.github>astrostl</bitbar.author.github>
+# <bitbar.desc>Checks for active turns at http://boiteajeux.net every two minutes</bitbar.desc>
+# <bitbar.image></bitbar.image>
+# <bitbar.dependencies></bitbar.dependencies>
+
+# INSTRUCTIONS: set baj_login=LOGIN and baj_password=PASSWORD in ~/.bitbar
+
+. ~/.bitbar
```

Comment:
Responded in that thread. I think variables-in-scripts is an antipattern, especially for GitHub projects :)

Very happy to refactor into any env or config file system that supports out-of-repo key/value storage!